src/components/invite/Invite.js

```diff
@@ -360,12 +359,11 @@ const Invite = () => {
   }
 
   useEffect(() => {
-    //reset state for rewards icon in navbar
-    if (inviteState.pending || inviteState.approved) {
-      userStorage.userProperties.set('lastInviteState', inviteState)
-      AsyncStorage.setItem('GD_lastInviteState', inviteState)
+    // reset state for rewards icon in navbar
+    if (initialized) {
+      userStorage.userProperties.set('lastInviteState', inviteState || { pending: 0, approved: 0 })
```

Comment:
resetting state for reward **only** after hook completed loading invitees and rewards data